tasks/files-to-granules/README.md

```diff
@@ -0,0 +1,46 @@
+# @cumulus/files-to-granules
```

Comment:
This README is 💯

Reply:
I feel like we should move the content in 858, merge styles a bit and auto-generate the task documentation or something.    Moving it all to the readme makes it not searchable, but.... There Must Be A Better Way.

Reply:
Noted. Will file a ticket to improve docs with task READMEs. Probably PI material.